spec/services/authentication/providers_spec.rb

```diff
@@ -32,10 +34,6 @@
   end
 
   describe ".enabled" do
-    it "lists all available providers" do
-      expect(described_class.available).to eq(described_class.enabled)
-    end
-
```

Comment:
why did we lose this?

Reply:
@rhymes Good question! 😄  In my mind, this spec made sense when we had all providers enabled as default. But that is no longer the case, so comparing `described_class.available` to `described_class.enabled` seems moot? Unless I am misunderstanding... please let me know

Reply:
Oh! you're right! :D Sorry, I got confused hahaha